Gemfile.jruby-3.1.lock.release

```diff
@@ -94,9 +95,11 @@ GEM
     clamp (1.0.1)
     coderay (1.1.3)
     concurrent-ruby (1.1.9)
-    crack (0.4.5)
+    crack (0.4.6)
+      bigdecimal
       rexml
-    dalli (3.2.6)
+    dalli (3.2.7)
+      base64
```

Comment:
hm,,,, we may partially backport [this](https://github.com/elastic/logstash/pull/15867/files)